ClientLib/src/main/java/com/o3dr/services/android/lib/drone/mission/MissionItemType.java

```diff
@@ -113,6 +114,18 @@ public MissionItem getNewItem() {
         }
     },
 
+    DO_LAND_START("Do Land start") {
```

Comment:
@ne0fhyk: For what purpose? The autopilot of copter/rover will simply ignore unsupported mission items.